CMakeLists.txt

```diff
@@ -33,7 +33,6 @@ option(OPENGEODE_GEOSCIENCESIO_WITH_PYTHON "Compile Python bindings" OFF)
 find_package(OpenGeode REQUIRED CONFIG)
 find_package(OpenGeode-IO REQUIRED CONFIG)
 find_package(GDAL REQUIRED CONFIG)
-find_package(OpenGeode-IO REQUIRED CONFIG)

```

Comment:
No keep it to ease cmake message error. Way simpler to understand "missing OpenGeode-IO" than "missing pugixml"